third_party/iap_jwt_verify_nginx/src/BUILD

```diff
@@ -0,0 +1,160 @@
+# Copyright (C) 2002-2016 Igor Sysoev
+# Copyright (C) 2011-2016 Nginx, Inc.
+# Copyright (C) 2017 Google Inc.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+# 1. Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE AUTHOR AND CONTRIBUTORS ``AS IS'' AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED.  IN NO EVENT SHALL THE AUTHOR OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS
+# OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+# HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+# OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+# SUCH DAMAGE.
+#
+################################################################################
+
+gtest_include_copt = (
+    "-Iexternal/gtest/googletest-release-1.8.0/googletest/include")
+
+cc_library(
+    name = "ngx_iap_jwt_verify",
+    srcs = [
+        "module.h",
```

Comment:
This target must have public visibility (it's what consumers of this module must link against). See nginx_modules.c under test/ (and the nginx-iap cc_binary there) for an example of how this is done.

Consumers don't need to include module.h, however, so I put it under srcs instead of hdrs. ESP does it the same way:
https://github.com/cloudendpoints/esp/blob/master/src/nginx/BUILD#L89